app/models/concerns/user_encrypted_attribute_overrides.rb

```diff
@@ -62,17 +64,25 @@ def stale_encrypted_email?
     encrypted_attributes[:email].stale?
   end
 
-  def phone
-    get_encrypted_attribute(name: :phone, default: nil)
+  ENCRYPTED_ATTRIBUTES.each do |attribute|
```

Comment:
This meta-programming means that the methods are not greppable. Eg: I see a use of `stale_encrypted_phone?` and grep for `def stale_encrypted_phone?` but it is nowhere to be found. Plus, since this is in a module rather than in `User`, it is even more likely that I will have difficulty finding this. 

This is the case whenever there is meta-programming, so not specific to implementation here. I think the trade-off is worth it when there are a bunch of attributes, but right now it's just 2 attributes that we're encrypting. Yes, we will need to change 2 separate methods if the implementation changes, but I think that overhead is the lesser evil. If we start encrypting more than 2 attributes, would be worth it to re-visit meta-programming for these methods, in my opinion. 

Curious what others think!